website/content/commands/connect/ca.mdx

```diff
@@ -34,8 +35,8 @@ Usage: consul connect ca <subcommand> [options] [args]
   For more examples, ask for subcommand help or view the documentation.
 
 Subcommands:
-    get-config    Display the current Connect Certificate Authority (CA) configuration
-    set-config    Modify the current Connect CA configuration
+    get-config    Display the current service mesh Certificate Authority (CA) configuration
```

Comment:
Note: I intend to submit a separate PR to change the CLI command help descriptions output by the code itself

Reply:
CLI help text update PR submitted for review by eng: https://github.com/hashicorp/consul/pull/17230